compiler/damlc/daml-ide-core/test/Development/IDE/State/API/Testing.hs

```diff
@@ -10,9 +10,12 @@
 -- | Testing framework for Shake API.
 module Development.IDE.Core.API.Testing
     ( ShakeTest
+    , TemplateProp(..)
+    , ExpectedChoices(..)
     , GoToDefinitionPattern (..)
     , HoverExpectation (..)
     , D.DiagnosticSeverity(..)
+    , expectedPoperties
```

Comment:
```suggestion
    , expectTemplateProperties
```
Matches the naming scheme of the other `expect*` functions.
I would also move it down in the export list to be next to those.

Reply:
Yup makes sense. Done